src/components/fields/checkbox-group/types.ts

```diff
@@ -12,14 +12,18 @@ interface IToggleOption extends IOption {
 	none?: boolean;
 }
 
+export type checkboxToggleLayoutType = "horizontal" | "vertical";
```

Comment:
convention is to append `T` to types

```suggestion
export type TCheckboxToggleLayoutType = "horizontal" | "vertical";
```